docs/managed-datahub/slack/saas-slack-app.md

```diff
@@ -0,0 +1,15 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# The Slack App
+
+<FeatureAvailability saasOnly />
+
+## Overview
+The DataHub Slack App brings several of DataHub's key capabilities directly into your slack experience. For instance, you can search for an asset directly within slack by simply typing `/acryl search revenue_data`.
```

Comment:
You actually don't need the "search" part :) 

Let's put this inside a top-level section called: "Searching for data assets" 

The "Overview" section can give a brief overview of the QUERY (search) and NOTIFICATION (subscription) use cases supported by Slack

We should also document that you can subscribe to assets via Slack. And we can also add a "Manaing data incidents" bit in here that describes how to resolve and reopen incidents via Slack. I can send you some screenshots there as well